src/cargo/ops/cargo_config.rs

```diff
@@ -4,20 +4,25 @@ use crate::util::config::{Config, ConfigKey, ConfigValue as CV, Definition};
 use crate::util::errors::CargoResult;
 use crate::{drop_eprintln, drop_println};
 use anyhow::{bail, format_err, Error};
+use clap::{ArgEnum, PossibleValue};
 use serde_json::json;
 use std::borrow::Cow;
 use std::fmt;
 use std::str::FromStr;
 
+#[derive(clap::ArgEnum, Clone)]
 pub enum ConfigFormat {
```

Comment:
Nit: This should probably be `Copy` as well.

Reply:
thanks for spotting! I had a vague feeling of inconsistency between the derives in ConfigFormat and Prefix